style.css

```diff
@@ -594,3 +574,351 @@ a {
+  }
+
+  .refer-link {
+    font-family: "Poppins", sans-serif;
+    font-weight: 500;
+    font-size: 16px;
+    line-height: 24px;
+    color: #7f8cff;
+  }
+
+  #about-container .about-content-box .icon-content-element {
+    width: 40%;
+    margin-top: 20px;
+    display: flex;
+    justify-content: space-between;
+    align-items: center;
+    transition: transform 0.05s ease-out;
+  }
+
+  #contact-container {
+    background-image: url(./images/contact_backward.png);
+    background-size: cover;
+    background-repeat: no-repeat;
+    z-index: 2;
+    margin: -120px auto;
+    padding: 100px 20px 0;
+    height: 112vh;
+  }
+
+  #description-contact-box {
+    width: 90%;
+    margin: auto;
+    height: 200px;
+    display: flex;
+    flex-direction: column;
+    justify-content: center;
+    align-items: center;
+    margin-bottom: 50px;
+  }
+
+  #description-contact-box .contact-title-1 {
+    font-family: "Poppins", sans-serif;
+    font-weight: bold;
+    font-size: 40px;
+    line-height: 52px;
+    color: white;
+    margin-bottom: 20px;
+  }
+
+  #description-contact-box .contact-description-1 {
+    font-family: "Poppins", sans-serif;
+    font-size: 20px;
+    line-height: 28px;
+    text-align: center;
+    color: #ebebff;
+    margin-bottom: 30px;
+    user-select: none;
+    cursor: pointer;
+    transition: all 0.2s ease-in-out;
+  }
+
+  #description-contact-box .contact-description-1:hover {
+    transform: translateX(10px);
+  }
+
+  #contact-container #input-contact-box {
+    height: 500px;
+    display: flex;
+    flex-direction: column;
+    justify-content: space-between;
+    margin-bottom: 20px;
+  }
+
+  #contact-container .label {
+    font-family: "Poppins", sans-serif;
+    color: white;
+    margin: 10px 0 5px;
+  }
+
+  #contact-container #input-contact-box .input-i {
+    font-family: "Poppins", sans-serif;
+    height: 48px;
+    padding: 10px;
+    font-size: 17px;
+    line-height: 20px;
+    color: #172b4d;
+    border: 1px solid #dcd6cf;
+    border-radius: 8px;
+    margin-bottom: 10px;
+    outline: none;
+  }
+
+  #contact-container #input-contact-box .input-m {
+    font-family: "Poppins", sans-serif;
+    padding: 5px 5px 0;
+    font-size: 15px;
+    line-height: 24px;
+    color: #787879;
+    border: 1px solid #cfd8dc;
+    border-radius: 8px;
+    outline: none;
+    letter-spacing: 2px;
+  }
+
+  #contact-container .get-touch-btn {
+    font-family: "Poppins", sans-serif;
+    font-weight: 500;
+    font-size: 17px;
+    line-height: 24px;
+    color: #6070ff;
+    width: 140px;
+    height: 58px;
+    left: 24px;
+    margin-top: 20px;
+    background: #fff;
+    border-radius: 8px;
+    border: none;
+  }
+
+  #contact-container .get-touch-btn:hover {
+    background: #6070ff;
+    box-shadow: 0 8px 16px rgb(64 83 252 / 24%);
+    border: 1px solid #fff;
+    color: #fff;
+  }
+
+  #contact-container .get-touch-btn:active {
+    background: #2230d2;
+    border: none;
+  }
+}
```

Comment:
- [ ] From the project requirements, you should stick to the [Figma design](https://www.figma.com/file/l7SqJ3ZfkAKih9sFxvWSR4/Microverse-Student-Project-1?node-id=39%3A122&t=Tt1vCN6lfPrkflxz-0), But as you can see there are some differences between the [Figma design](https://www.figma.com/file/l7SqJ3ZfkAKih9sFxvWSR4/Microverse-Student-Project-1?node-id=39%3A122&t=Tt1vCN6lfPrkflxz-0) and your design
  - From the `textarea` field to the bottom of the page does not have background color as the figma design. 

| Your Design | Figma Design |
| :---:   | :---: | 
|![image](https://user-images.githubusercontent.com/92303891/220604868-1953dc03-e93b-40e0-ba78-1dfdcfb3141b.png)| ![image](https://user-images.githubusercontent.com/92303891/220605467-a6322dfa-dc79-488f-905f-b0fe56d89b76.png)|